components/fxa-client/Cargo.toml

```diff
@@ -15,7 +15,7 @@ hawk = "1.0.5"
 hex = "0.3.2"
 lazy_static = "1.0.0"
 log = "0.4"
-openssl = { version = "= 0.10.20", optional = true }
+openssl = { version = "= 0.10.23", optional = true }
```

Comment:
I guess we can remove the `=` prefixes now since they have fixed the bug in openssl.